src/clients/rust/README.md

```diff
@@ -136,13 +166,48 @@ See all error conditions in the [create_accounts
 reference](https://docs.tigerbeetle.com/reference/requests/create_accounts).
 
 ```rust
-// // todo
+let account0 = tb::Account {
+    id: (102),
+    ledger: 1,
+    code: 718,
+    ..Default::default()
+};
+let account1 = tb::Account {
+    id: (103),
+    ledger: 1,
+    code: 718,
+    ..Default::default()
+};
+let account2 = tb::Account {
+    id: (104),
+    ledger: 1,
+    code: 718,
+    ..Default::default()
+};
+
+let account_errors = client
+    .create_accounts(&[account0, account1, account2])
+    .await?;
+
```

Comment:
Can we work in `assert!(account_errors.len() <= 3)`? This clarifies semantics somewhat (that you get errors only for failures, and don't need zipping).